curious/commands/cmd.py

```diff
@@ -97,6 +98,7 @@ def __init__(self, cbl, *,
             # This isn't always accurate, but you should provide `name=` if you want that.
             self.name = self.callable.__name__
 
+        self.description = description or inspect.getdoc(self.callable)
```

Comment:
The `or inspect.getdoc` should be inside `get_help`, this should just be self.description